### PR TITLE
Add instructions how to build the development environment with podman

### PR DESCRIPTION
**Pull request by dcermak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,16 @@ development environment you can use.
    docker-compose run --rm obs-docu daps -vv -d DC-obs-all html
    ```
 
+### Podman based Development Environment
```
Comment (dmarcoux) [inline README.md:49]:
```suggestion
### Podman-based Development Environment
```

**Review by dmarcoux — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -1,11 +1,13 @@
 FROM openbuildservice/obs-docu
 ARG CONTAINER_USERID
+ARG USER=frontend
 
 # Configure our user id
-RUN usermod -u $CONTAINER_USERID frontend
+ENV CONTAINER_USERID=$CONTAINER_USERID
```
Comment (hennevogel) [inline Dockerfile:6]: Why write it to the env?

**Comment by dcermak** [comment]
Henne Vogelsang ***@***.***> writes:

> @hennevogel commented on this pull request.
>
>
>
>>  
>  # Configure our user id
> -RUN usermod -u $CONTAINER_USERID frontend
> +ENV CONTAINER_USERID=$CONTAINER_USERID
>
> Why write it to the env?

I thought it was necessary for the RUN command further down to work
properly.

**Comment by hennevogel** [comment]
> I thought it was necessary for the RUN command further down to work properly.

Nope, see https://docs.docker.com/engine/reference/builder/#arg

**Comment by dcermak** [comment]
Henne Vogelsang ***@***.***> writes:

>> I thought it was necessary for the RUN command further down to work properly.
>
> Nope, see https://docs.docker.com/engine/reference/builder/#arg

You're right, I've removed that line then.

**Review by hennevogel — APPROVED** [review]

**Pull request merged**